pkg/collector/corechecks/snmp/internal/devicecheck/devicecheck.go

```diff
@@ -34,6 +34,7 @@ import (
 const (
 	snmpLoaderTag        = "loader:core"
 	serviceCheckName     = "snmp.can_check"
+	deviceUpMetric       = "snmp.device_up"
```

Comment:
Do you mind giving a bit more details about "Tracking device stability over time.", what do you mean by stability?